core/templates/dev/head/css/third_party_override.css

```diff
@@ -0,0 +1,23 @@
+.select2-container,
+.oppia-html-select{
+	width: 60%;
+}
+.select2-container .select2-choice{
+	width: 100%;
+	height: 36px;
+	padding: 4px 0 0 8px;
+}
+.select2-container .select2-choice,
+.select2-results .select2-result-label,
+.oppia-html-select .oppia-html-select-selection-element,
+.oppia-html-select .oppia-html-select-option{
+	text-align: left;
+	font-size: 14px;
+}
+.oppia-html-select .oppia-html-select-selection{
+	width: 100%;	
+}
+.dropdown-menu{
+	width: 100% !important;
```

Comment:
"!important" was used for overriding inline css of core/templates/dev/head/components/html_select_directive.html at line 7. There is no need to write inline css (width:inherit ). Now I am removing that as it will be in separate file (i.e. third_party_override.css)